src/plugins/intel_gpu/src/runtime/ocl/ocl_engine.cpp

```diff
@@ -87,6 +87,21 @@ const cl::UsmHelper& ocl_engine::get_usm_helper() const {
     return *_usm_helper;
 }
 
+allocation_type ocl_engine::detect_allocation_type(const void* memory) const {
+    ocl::gpu_usm::detect_allocation_type(this, memory);
```

Comment:
Why don't you just return result of this call?

Reply:
It was original intention, fixed, thanks